### ISSUE TITLE
Make `]x` and `[x` movements wrap around the end of file

### ISSUE THREAD

**Issue by axlebedev** [rhysd/conflict-marker.vim#6]
Really need this feature

**Comment by hayderhassan** [comment]
I agree, this is an important feature. I see it hasn't been updated yet so I've made a Pull Request: https://github.com/rhysd/conflict-marker.vim/pull/23 

@rhysd please could you kindly review it and approve if you're happy?
